prow/deck/deployment.yaml

```diff
@@ -23,12 +22,13 @@ spec:
       terminationGracePeriodSeconds: 30
       containers:
         - name: deck
-          image: gcr.io/k8s-prow/deck:v20190307-bbad344
+          image: gcr.io/k8s-prow/deck:v20190409-ef11deceb
+          imagePullPolicy: Always
```

Comment:
Thoughts on adding to all?

Reply:
I would assume the tags are immutable so I'm not sure it's necessary, will slow down container starts a bit right? I'm tempted to leave until it causes a problem 😉